### ISSUE TITLE
How to export my data from SwiftnessX

### ISSUE THREAD

**Issue by jinyu00** [ehrishirajsharma/SwiftnessX#10]
Where is my data store in?

**Comment by ehrishirajsharma** [comment]
for which OS?

**Comment by ehrishirajsharma** [comment]
On bottom left, there is an option (beside the Swiftness Logo), click on it and you would be able to export/import the data..

If you're asking where your data stored locally then let me know your OS..

**Comment by jinyu00** [comment]
It seem the data store in 
```
C:\Users\xxxx\AppData\Roaming\Swiftness\storage
```

It just store data in one json file? if the data is big , the tools may be slow?

**Comment by ehrishirajsharma** [comment]
Although, we have tested it with enough (from regular workflow) - it didn't much affect the performace however, this can be improved with tree.

We intentionally kept the storage under JSON file to let users easily understand the schema and make other tools easily integratable..however we will look into the performance section..

**Comment by hakluke** [comment]
It would be good to be able to choose where to store the data. That way it would be trivial to backup to cloud storage - Dropbox/Google Drive, etc. and use across multiple machines without native cloud support. This could be implemented with basic File->Open and File->Save/Save As functionality.

**Comment by ehrishirajsharma** [comment]
Yes, we have achieved this partially and have this on our roadmap: https://github.com/ehrishirajsharma/SwiftnessX/wiki/Upcomings

We just want to deploy better versioning control (vs auto sync) - giving an option to pull the changes rather making it auto-sync so even teams could efficiently work on a project. 

We may be rolling out this on our private-beta very soon, let me know if you're interested on joining this.

**Comment by hakluke** [comment]
I'm definitely interested in the private beta for this.

**Comment by mpgn** [comment]
Hello @ehrishirajsharma 

Can't find where the json file is stored on linux, can you provide the default path ?

**Comment by machinexa2** [comment]
.config/Electron/storage/content.json is on this place

**Comment by machinexa2** [comment]
Also if you want to find where its located then take a paragraph from your swiftness x location  and do this
```
cd /home
rgrep -i "some line, single line will do the job but make your you copied correctly and exactly" 
cd /root
rgrep -i "some line, single line will do the job but make your you copied correctly and exactly" 
```

**Comment by machinexa2** [comment]
Also if you want to find where its located then take a paragraph from your swiftness x location  and do this
```
cd /home
rgrep -i "some line, single line will do the job but make your you copied correctly and exactly" 
cd /root
rgrep -i "some line, single line will do the job but make your you copied correctly and exactly" 
```
